src/main/java/com/hida/DatabaseManager.java

```diff
@@ -0,0 +1,261 @@
+        try {
+            // connect to database
+            Class.forName("org.sqlite.JDBC");
+            DATABASE_CONNECTION = DriverManager.getConnection(DRIVER);
+            if (!isDbSetup()) {
+                System.out.println("creating table");
+
+                // string to hold a query that sets up table in SQLite3 syntax
+                String sqlQuery = String.format("CREATE TABLE %s "
+                        + "(%s PRIMARY KEY NOT NULL);", TABLE_NAME, COLUMN_NAME);
+
+                // a database bus that allows database/webservice communication 
+                Statement databaseBus = DATABASE_CONNECTION.createStatement();
+
+                // create non-existing table and clean-up
+                databaseBus.executeUpdate(sqlQuery);
+                databaseBus.close();
+            }
+            return true;
+        } catch (SQLException exception) {
+            System.err.println(exception.getMessage());
+            return false;
+        } catch (ClassNotFoundException exception) {
+            System.err.println(exception.getMessage());
+            return false;
+        }
+
+    }
+
+    /**
+     * Adds a requested amount of formatted ids to the database.
+     *
+     * @param list - list of ids to check.
+     */
+    public synchronized void addId(Set<String> list) {
+        //ArrayList<String> redundantIdList = new ArrayList();
+        try {
+
+            // a string to hold sqlite3 queries
+            String sqlQuery;
+
+            for (String id : list) {
+                // assign a sqlite3 query that adds an id to the database
+                System.out.println("adding id...");
+                sqlQuery = String.format("INSERT INTO %s (%s) "
+                        + "VALUES('%s')", TABLE_NAME, COLUMN_NAME , id);
+
+                // a statement that allows database/webservice communication
+                Statement insertStatement
+                        = DATABASE_CONNECTION.createStatement();
+
+                // execute query
+                insertStatement.executeUpdate(sqlQuery);
+                System.out.println("id added");
+
+                // clean-up                
+                insertStatement.close();
+            }
+
+        } catch (SQLException exception) {
+            System.err.println(exception.getMessage() + " in addId");
+            System.err.println(Arrays.toString(exception.getStackTrace()));
+        }
+        System.out.println("printed to database");
+        // prints out to server what ids were created
+        //printData();
+        
+    }
+
+    /**
+     * Checks a list of ids against a database to ensure that no redundancies
+     * are added.
+     *
+     * @param list - list of ids to check.
+     * @return - a list containing ids that already exist in database. Returns
+     * an empty list of all the ids given in the param list are unique.
+     */
+    public Set<String> checkId(Set<String> list) {

```

Comment:
I think I mentioned elsewhere, but this should be named something more descriptive and selfd-documenting, like `findExistingIds`